lib/models/wallet-v2.js

```diff
@@ -262,11 +262,26 @@ walletV2.clearTotpRemovalRequestIfPossible = function(wallet) {
   if(!wallet.totpDisabledAt) {
     return Promise.resolve();
   } else {
-    // set the totpDisabledAt to null
+    var now        = Math.floor((new Date()).getTime() / 1000);
+    var disabledAt = Math.floor(wallet.totpDisabledAt.getTime() / 1000);
```

Comment:
Hmmm, I copied it from [here](https://github.com/stellar/stellar-wallet/blame/fix-62/lib/models/wallet-v2.js#L294). I was thinking about the same thing but I thought there's something important behind rounding it to seconds. Can change this.